test_scripts/API/VehicleData/StabilityControlsStatus/004_OnVehicleData_StabilityControlsStatus.lua

```diff
@@ -0,0 +1,114 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check receiving StabilityControlsStatus data via OnVehicleData notification
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+-- 5) App is subscribed on StabilityControlsStatus vehicle data and is not subscribed on GPS data
+
+-- Steps:
+-- 1) HMI sends VehicleInfo.OnVehicleData notification with StabilityControlsStatus data
+--   (escSystem = "ON", trailerSwayControl = "OFF")
+-- SDL does:
+--  - send OnVehicleData notification with received from HMI data to App
+-- 2) HMI sends VehicleInfo.OnVehicleData notification with GPS data
```

Comment:
Test case steps does not reflect completely test script scenario. Please correct steps description

Reply:
@VjKlepikov Fixed (see a7a20fe)